sdformat_to_mjcf/sdformat_to_mjcf/converters/link.py

```diff
@@ -79,7 +80,10 @@ def add_link(body, link, parent_name="world"):
 
     for vi in range(link.visual_count()):
         vis = link.visual_by_index(vi)
+        material = None
         if vis.geometry() is not None:
-            add_visual(body, vis)
+            visual_geom = add_visual(body, vis)
+            if vis.material() is not None:
+                material = add_material(visual_geom, vis.material())
```

Comment:
nit: no need to assign `material`.

Reply:
https://github.com/gazebosim/gz-mujoco/pull/34/commits/aabe6d77327040b4a38282b15087a3a6b3725abb